applications/TrilinosApplication/custom_python/trilinos_pointer_wrapper.h

```diff
@@ -163,7 +163,7 @@ class AuxiliaryMatrixWrapper
     ///@}
 }; // Class AuxiliaryMatrixWrapper
 
-/// Auxiliarty Trilinos vector pointer wrapper class
```

Comment:
your replace is wrong here

Reply:
Done

(Comments better in the changed part to track changes)